call-of-duty4mwconfig.json

```diff
@@ -0,0 +1,478 @@
+[
+    {
+        "DisplayName":"Game Port",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"$GamePort",
+        "InputType":"number",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"net_port",
+        "IncludeInCommandLine":false,
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"IP Binding",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"$ApplicationIPBinding",
+        "InputType":"number",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"net_ip",
+        "IncludeInCommandLine":false,
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Enable Log File",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"logfile",
+        "InputType":"number",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"logfile",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"1",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Continuous Log",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"g_logsync",
+        "InputType":"number",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"g_logsync",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"2",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Log File Name",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"g_log",
+        "InputType":"text",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"g_log",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"games_mp.log",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Disable Punkbuster",
+        "Category":"Server Settings",
+        "Description":"",
+        "Keywords":"",
+        "FieldName":"sv_punkbuster",
+        "InputType":"number",
+        "Hidden":true,
+        "IsFlagArgument":false,
+        "ParamFieldName":"sv_punkbuster",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"0",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Server Name",
+        "Category":"Server Settings",
+        "Description":"Sets the name of the server. Colour codes can be used: ^0=Black, ^1=Red, ^2=Green, ^3=Yellow, ^4=Blue, ^5=Cyan, ^6=Pink, ^7=White, ^8=Grey, ^9=Grey/Map Default",
+        "Keywords":"server,name",
+        "FieldName":"sv_hostname",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"sv_hostname",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"^1AMP Powered Call of Duty 4: Modern Warfare Server",
```

Comment:
The server name gets cut off after `AMP Powered Call of Du`. I'd suggest shortening it and noting the character limit in the description.

Reply:
Is that just a steam browser issue or does it also happen in-game?

Reply:
This is an in-game limitation. Every server in the list is that many characters or less.

Reply:
OK. So 22 chars, or 24 including the colour formatting?

Reply:
Sorry for the slow response. 22 characters + 2 color formatting or 22 characters without color formatting. 22 is all that will be displayed in the browser.

Reply:
Updated.